test/circuits_sim/device/bmp280_test.exs

```diff
@@ -0,0 +1,12 @@
+defmodule CircuitsSim.Device.BMP280Test do
+  use ExUnit.Case
+
+  test "simple usage" do
+    {:ok, bmp_pid} = BMP280.start_link(bus_name: "i2c-1", bus_address: 0x77)
+    assert :ok = BMP280.force_altitude(bmp_pid, 100)
+    assert {:ok, measurement} = BMP280.measure(bmp_pid)
+    assert measurement.temperature_c == 26.74596725741867
+    assert measurement.humidity_rh == 59.23373947597236
+    assert measurement.pressure_pa == 100_391.49177320654
```

Comment:
I think this won't be a problem with Elixir, but I worry when comparing floats for equality. I always use `assert_in_delta` like below. I trimmed the precision pretty severely to guess what the sensor's precision really was, but I don't know.

```suggestion
    assert_in_delta measurement.temperature_c, 26.7, 0.1
    assert_in_delta measurement.humidity_rh == 59.2, 0.1
    assert_in_delta measurement.pressure_pa == 100_391, 1
```